mobile/src/main/res/values/strings.xml

```diff
@@ -18,15 +18,11 @@
     <string name="mainmenu_openhab_sitemaps">Sitemaps</string>
     <string name="mainmenu_openhab_preferences">Settings</string>
     <string name="mainmenu_openhab_habpanel">HABPanel</string>
-    <string name="mainmenu_openhab_habpanel_on_server">HABPanel (%s)</string>
-    <string name="mainmenu_openhab_oh3_ui">openHAB 3 UI</string>
-    <string name="mainmenu_openhab_oh3_ui_on_server">openHAB 3 UI (%s)</string>
+    <string name="mainmenu_openhab_oh3_ui">Main UI</string>
     <string name="mainmenu_openhab_frontail">Frontail</string>
-    <string name="mainmenu_openhab_frontail_on_server">Frontail (%s)</string>
     <string name="app_notifications">Notifications</string>
-    <string name="app_notifications_on_server">Notifications (%s)</string>
```

Comment:
I am not sure removing those strings in favor of hardcoding the server name position is a good idea. Some languages might want to change the order or replace the parentheses by something else.